src/PyListProxyHandler.cc

```diff
@@ -621,7 +620,7 @@ static bool array_lastIndexOf(JSContext *cx, unsigned argc, JS::Value *vp) {
   }
 
   JS::RootedValue elementVal(cx, args[0].get());
-  PyObject *element = pyTypeFactory(cx, elementVal)->getPyObject();
+  PyObject *element = pyTypeFactory(cx, elementVal);
```

Comment:
`element` needs to be `Py_DECREF`'d before the end of this function call

Reply:
done